xs2a-adapter-service-api/src/main/java/de/adorsys/xs2a/adapter/service/PropertyUtil.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018-2018 adorsys GmbH & Co KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.adorsys.xs2a.adapter.service;
+
+public final class PropertyUtil {
+
+    private PropertyUtil() {}
+
+    public static String readProperty(String key) {
+        return readProperty(key, "");
+    }
+
+    public static String readProperty(String key, String defaultValue) {
```

Comment:
What do you think about rewriting this method as following to omit extra check for not empty property value?
```
        String property = System.getProperty(key, "");
        if (property.isEmpty()) {
            property = System.getenv(key);
            return property == null || property.trim().isEmpty() ? defaultValue : property;
        }
        return property;
```

OR

```
        String property = System.getProperty(key, "");
        if (!property.isEmpty()) {
            return property;
        }
        property = System.getenv(key);
        return property == null || property.trim().isEmpty() ? defaultValue : property;
```

Reply:
thanks, fixed